packages/gatsby/src/schema/types/__tests__/date.js

```diff
@@ -3,7 +3,8 @@ const { actions } = require(`../../../redux/actions`)
 const { build } = require(`../..`)
 const withResolverContext = require(`../../context`)
 const { defaultResolver } = require(`../../resolvers`)
-const { isDate, looksLikeADate } = require(`../date`)
+import { isDate, looksLikeADate } from "../date"
+require(`../../../db/__tests__/fixtures/ensure-loki`)()
```

Comment:
hmm 🤔 
I think I missed the rebase. I look into this.

Reply:
I think you can drop all changes to this file except for the actual `import` change ;)

Reply:
After formatbot, the require just needs to be dropped (loki was removed). Thanks!

Reply:
(Rebase bot did not fix this, you'll have to do it)